src/test/regress/expected/sqlancer_failures.out

```diff
@@ -416,11 +416,11 @@ ON (true);
 SELECT
  COUNT(unsupported_join.*)
 FROM
-   (distributed_table a
+   ((distributed_table a
```

Comment:
can we merge this to the main branch directly? If so, can you open a separate PR?

Reply:
I will open a PR with 3 test changes